scripts/index.js

```diff
@@ -0,0 +1,263 @@
+const state = {
+  tempClickCount: 65,
+};
+
+/////////////////////////////////
+//  Temperature Color Condition//
+/////////////////////////////////
+const getColorClass = (temp) => {
+  if (temp <= 49) {
+    return "freezing";
+  }
+
+  if (temp <= 59) {
+    return "cool";
+  }
+
+  if (temp <= 69) {
+    return "sunlight";
+  }
+
+  if (temp <= 79) {
+    return "warm";
+  }
+
+  return "hot";
+}
+
+/////////////////////////////////////
+// Landscape Temperature Condition //
+////////////////////////////////////
+const getLandscapes= (temp) => {
+  if (temp <= 59) {
+    return cools;
+  }
+
+  if (temp <= 69) {
+    return wind;
+  }
+
+  if (temp <= 79) {
+    return rains;
+  }
+
+  return suns;
+}
+
+/////////////////////////////////////
+// Picture Change  Condition //
+////////////////////////////////////
+const getImage= (temp) => {
```

Comment:
Notice that `getImage` and `getLandscapes` have lots of the same code. Consider how you could refactor and combine this functionality to DRY up your code.